lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -361,5 +291,11 @@ if Enum.all?(
           |> to({:hls_sink_bin, track.stream_id})
         ]
       }
+
+    defp get_depayloader(track) do
+      track
+      |> Track.get_depayloader()
+      |> tap(&unless &1, do: raise("Couldn't find depayloader for track #{inspect(track)}"))
```

Comment:
Not sure if more readable, but you may consider:
```suggestion
      |> Track.get_depayloader() ||
        raise "Couldn't find depayloader for track #{inspect(track)}"
```
I'm surprised it's formatted this way